Emby.Server.Implementations/HttpClientManager/HttpClientManager.cs

```diff
@@ -324,125 +302,84 @@ private async Task<HttpResponseInfo> SendAsyncInternal(HttpRequestOptions option
-                    await client.SendAsync(httpWebRequest).ConfigureAwait(false);
+                if (options.RequestContentBytes != null)
+                {
+                    httpWebRequest.Content = new ByteArrayContent(options.RequestContentBytes);
+                }
+                else if (options.RequestContent != null)
+                {
+                    httpWebRequest.Content = new StringContent(options.RequestContent);
+                }
+                else
+                {
+                    httpWebRequest.Content = new ByteArrayContent(Array.Empty<byte>());
                 }
-                catch (Exception ex)
+                /*
+                var contentType = options.RequestContentType ?? "application/x-www-form-urlencoded";
+
+                if (options.AppendCharsetToMimeType)
                 {
-                    throw new HttpException(ex.Message) { IsTimedOut = true };
+                    contentType = contentType.TrimEnd(';') + "; charset=\"utf-8\"";
                 }
+
+                httpWebRequest.Headers.Add(HeaderNames.ContentType, contentType);*/
             }
 
             if (options.LogRequest)
             {
                 _logger.LogDebug("HttpClientManager {0}: {1}", httpMethod.ToString(), options.Url);
             }
 
-            try
-            {
-                options.CancellationToken.ThrowIfCancellationRequested();
-
-                /*if (!options.BufferContent)
-                {
-                    var response = await client.HttpClient.SendAsync(httpWebRequest).ConfigureAwait(false);
-
-                    await EnsureSuccessStatusCode(client, response, options).ConfigureAwait(false);
-
-                    options.CancellationToken.ThrowIfCancellationRequested();
-
-                    return GetResponseInfo(response, await response.Content.ReadAsStreamAsync().ConfigureAwait(false), response.Content.Headers.ContentLength, response);
-                }*/
-
-                using (var response = await client.SendAsync(httpWebRequest).ConfigureAwait(false))
-                {
-                    await EnsureSuccessStatusCode(response, options).ConfigureAwait(false);
-
-                    options.CancellationToken.ThrowIfCancellationRequested();
-
-                    using (var stream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false))
-                    {
-                        var memoryStream = new MemoryStream();
-                        await stream.CopyToAsync(memoryStream).ConfigureAwait(false);
-                        memoryStream.Position = 0;
+            options.CancellationToken.ThrowIfCancellationRequested();
 
-                        return GetResponseInfo(response, memoryStream, memoryStream.Length, null);
-                    }
-                }
-            }
-            catch (OperationCanceledException ex)
+            /*if (!options.BufferContent)
```

Comment:
Same here

Reply:
This was already there, didn't touch that afaik

Reply:
Maybe add a TODO to make it searchable?